src/components/review/ReviewConfiguration.jsx

```diff
@@ -79,78 +210,130 @@ const ReviewConfiguration = ({ idPrefix, setIsFormValid }) => {
+                                            <ChangeHostname
+                                              initHostname={initHostname}
+                                              setInitHostname={setInitHostname} />
+                                        </FlexItem>
+                                    </Flex>
+                                </DescriptionListDescription>
+                            </DescriptionListGroup>
+                        </ReviewDescriptionList>
+                    )}
+                </ReviewDescriptionList>
+            </FlexItem>
+            <FlexItem>
+                <ReviewDescriptionList>
+                    <ReviewDescriptionList>
+                        <DescriptionListGroup>
+                            <DescriptionListTerm>
+                                {_("Installation type")}
+                            </DescriptionListTerm>
+                            <DescriptionListDescription id={idPrefix + "-target-system-mode"}>
+                                {scenarioLabel}
+                            </DescriptionListDescription>
+                        </DescriptionListGroup>
+                    </ReviewDescriptionList>
+                    {storageScenarioId !== "mount-point-mapping" &&
+                    <ReviewDescriptionList>
+                        <DescriptionListGroup>
+                            <DescriptionListTerm>
+                                {_("Disk encryption")}
+                            </DescriptionListTerm>
+                            <DescriptionListDescription id={idPrefix + "-target-system-encrypt"}>
+                                {partitioning.method === "AUTOMATIC" && partitioning.requests[0].encrypted ? _("Enabled") : _("Disabled")}
+                            </DescriptionListDescription>
+                        </DescriptionListGroup>
+                    </ReviewDescriptionList>}
+                    <ReviewDescriptionList>
+                        <DescriptionListGroup>
+                            <DescriptionListTerm>
+                                {_("Storage")}
+                            </DescriptionListTerm>
+                            <DescriptionListDescription id={idPrefix + "-target-storage"}>
+                                <Stack hasGutter>
+                                    <StorageReview
+
+                                    />
+                                </Stack>
+                            </DescriptionListDescription>
+                        </DescriptionListGroup>
+                    </ReviewDescriptionList>
+                </ReviewDescriptionList>
+            </FlexItem>
+        </Flex>
     );
 };
 
-export const ReviewConfigurationConfirmModal = ({ idPrefix, onNext, setNextWaitsConfirmation }) => {
-    const { buttonLabel, buttonVariant, dialogTitleIconVariant, dialogWarning, dialogWarningTitle } = useScenario();
-
+export const ReviewConfigurationConfirmModal = ({
+    idPrefix,
```

Comment:
these changes are unrelated and add noise to the commit. Please avoid code-reorganization in feature commits.